specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -246,6 +247,7 @@ const processFieldType: {
             : getProperty('viewBtn')?.toLowerCase() === 'true',
         typeSearch: getProperty('name'),
         searchView: getProperty('searchView'),
+        defaultRecordId: getProperty('defaultRecordId'),
```

Comment:
(just a note) 

If desired, you can do whatever additional parsing here that you need! 

For example, you can check if the value here is one of the defaults and if not, then try and parse it as an integer and otherwise return undefined. 

That way, when working with `defaultRecordId` later down the line, you already know if it's supposed to be converted to a resource_uri, is one of the defaults, etc. 

If you do change how the defaultRecordId is parsed, don't forget to modify the type definition accordingly and other places it's used (like the syncer).